Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -122,6 +122,11 @@ def tests
       unless OS.mac?
         bundle_args << "--tag" << "~needs_macos" << "--tag" << "~cask"
         files = files.reject { |p| p =~ %r{^test/(os/mac|cask)(/.*|_spec\.rb)$} }
+
+        if HOMEBREW_PATCHELF_RB
+          # don't overwrite the gem environment setup for bundler
+          Homebrew.install_gem! "patchelf", version: Homebrew::PATCHELF_VERSION, setup_gem_environment: false
```

Comment:
I had one more thought of what could work: What about something like this in the `Gemfile`:

```ruby
if ENV["HOMEBREW_PATCHELF_RB"]
  gem "patchelf"
end
```

and then not commit any changes to e.g. `Gemfile.lock` and add it to `.gitignore`. Does that work as expected for you?

Reply:
We did that for https://github.com/Homebrew/brew/pull/7725 and it's worked pretty well there.

Reply:
thanks @MikeMcQuaid, i'll have it changed soon,  the ruby in [system is broke](https://github.com/Homebrew/homebrew-portable-ruby/issues/104), im trying work arounds  but im not sure if that is the way to do it. phew